src/ui/state/stakeholder.rs

```diff
@@ -96,6 +99,7 @@ impl State for StakeholderHomeState {
             ctx,
             None,
             Vec::new(),
+            self.display_deposit_card,
```

Comment:
```suggestion
            self.vaults.is_empty(),
```

Reply:
Actually, the StakeholderHomeState doesn't have a `vaults` field

Reply:
Ok can you please add one, we will use it to filter unvaulting vaults later